src/utilities/validateReport.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Check if the given parameter is a valid Report based
+ * on its data structure and type and return a boolean.
+ *
+ * @param   {object}  parsedData  The data to check.
+ * @param   {string}  type        Type of Report to check.
+ */
+
+import Ajv from 'ajv';
+
+// Expected json structure for Table reports
+const tableReportSchema = {
```

Comment:
I'd potentially be tempted to put a docstring on each schema which had a quick basic example of the schema - it's quite hard to read (and can't be helped) but for scanning.. i.e.:

```
{
    data: { 
        header: [...],
        rows: [...],
        formatters: [...],
    },
}
```

for quickly scanning, it's nice to have a little reference for the shape for any large object

Reply:
Done here 
e7779b1